app/src/main/java/org/memento/presentation/setting/component/SettingTopBar.kt

```diff
@@ -0,0 +1,109 @@
+package org.memento.presentation.setting.component
+
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.padding
+import androidx.compose.material3.Icon
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.alpha
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.graphics.vector.ImageVector
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.res.vectorResource
+import androidx.compose.ui.text.style.TextAlign
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import org.memento.R
+import org.memento.presentation.type.SettingTopBarType
+import org.memento.presentation.util.noRippleClickable
+import org.memento.ui.theme.MementoTheme
+import org.memento.ui.theme.darkModeColors
+
+/**
+ *  @param type Setting인지 TopBar인지 type구분
+ *  @param onBackClick 뒤로가기 눌렀을 떄
+ *  @param onDoneClick  Done 눌렀을 때
+ *  @param isDoneVisible Done 텍스트를 보일지 말지 선택
+ *  @param modifier modifier 수정 사항
+ */
+
+@Composable
+fun SettingTopBar(
+    type: SettingTopBarType,
+    onBackClick: () -> Unit,
+    modifier: Modifier = Modifier,
+    onDoneClick: () -> Unit,
```

Comment:
p2 :
```suggestion
    onDoneClick: () -> Unit = {},
```

이런식으로 해주깃..? Done이 없는 컴포넌트도 있어서요 !!